docs/changelog.txt

```diff
@@ -48,6 +48,7 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 ## Misc Improvements
 - Terminal console no longer appears in front of the game window on startup
 - `gui/design`: Improved performance for drawing shapes
+- ``Core``: For debugging purposes, you can now pass ``--disable-dfhack`` on the Dwarf Fortress commandline to disable DFHack for the session.
```

Comment:
Stylistic nitpick: this isn't a literal name, so I would drop the backticks:
```suggestion
- Core: For debugging purposes, you can now pass ``--disable-dfhack`` on the Dwarf Fortress commandline to disable DFHack for the session.
```
and as a bonus (or not, depending on your preferences), this would sort it closer to the top of the list

Reply:
ha ; ) fair enough.